get_info.php

```diff
@@ -1 +1,171 @@
-<?php?>
+<?php
+require 'db/config.php';
+require 'statements.php';
+
+// Handle client request error
+try {
+    $method = "";
+    if (isset($_SERVER['REQUEST_METHOD'])) {
+
+        $query = "";
+        $method = $_SERVER['REQUEST_METHOD'];
+        // Request stored as associative array
+        $data = json_decode(file_get_contents("php://input"), true);
+        switch ($method) {
+
+            case "POST":
+                $query = handle_post($data);
+                break;
+
+            case "GET":
+                $query = handle_get();
+                break;
+
+            default:
+                throw new Exception("Invalid Request");
+                break;
+        }
+    }else{
+        $error = 'No Request Method Found';
+        throw new Exception($error);
+    }
+} catch (Exception $e) {
+    $result = HTTPStatus(400);
+    date_default_timezone_set('America/Toronto');
+    error_log(date("Y-m-d h:i:sa")." : ".$e->getMessage()."\n", 3, "errors.log");
+    print json_encode($result, JSON_PRETTY_PRINT);
+    exit();
+}
+
+// Handle SQL error
```

Comment:
Are you handling a sql error, or are you attempting to get data via sql request?